espresso-server/app/src/androidTest/java/io/appium/espressoserver/lib/helpers/BackdoorUtils.java

```diff
@@ -1,53 +1,80 @@
 package io.appium.espressoserver.lib.helpers;
 
+import java.util.ArrayList;
 import java.util.List;
 
+import io.appium.espressoserver.lib.model.BackdoorMethodArgs;
+
 public class BackdoorUtils {
-    public static Class<?> mapToPrimitiveClass(Class<?> c) {
-        if (c.equals(Integer.class)) return int.class;
-        if (c.equals(Float.class)) return float.class;
-        if (c.equals(Double.class)) return double.class;
-        if (c.equals(Boolean.class)) return boolean.class;
-        if (c.equals(Long.class)) return long.class;
-        if (c.equals(Byte.class)) return byte.class;
-        if (c.equals(Short.class)) return short.class;
-        if (c.equals(Character.class)) return char.class;
-
-        return c;
-    }
 
-    public static Class<?> mapToBoxingClass(Class<?> c) {
-        if (c.equals(int.class)) return Integer.class;
-        if (c.equals(float.class)) return Float.class;
-        if (c.equals(double.class)) return Double.class;
-        if (c.equals(boolean.class)) return Boolean.class;
-        if (c.equals(long.class)) return Long.class;
-        if (c.equals(byte.class)) return Byte.class;
-        if (c.equals(short.class)) return Short.class;
-        if (c.equals(char.class)) return Character.class;
-
-        return c;
+    public static Object parseValue(Object o, Class<?> c) {
```

Comment:
do we want to also support array types like int[]?

Reply:
No. In fact, we do not need to support so many types. In my experience usually, only String is sufficient because the backdoor method will be defined by developers in such a way that it takes care of parsing